doc/release-notes.md

```diff
@@ -130,9 +136,6 @@ Changes to Wallet or GUI related settings can be found in the GUI or Wallet sect
 
 - Passing an invalid `-rpcauth` argument now cause bitcoind to fail to start.  (#20461)
 
-- The `getnodeaddresses` RPC now returns a "network" field indicating the
-  network type (ipv4, ipv6, onion, or i2p) for each address.  (#21594)
```

Comment:
This move is unrelated to this PR. If this PR does not fly, when the move will not fly either.

Reply:
I think it's ok. If this PR doesn't make the release, I can move the existing note to the right place when the notes are moved to the wiki for editing (I usually make some corrections in the wiki before the notes are moved back to the repo for -final).